www/src/config.js

```diff
@@ -130,9 +130,49 @@ const INSIGHTS_PAGES = [
   },
 ];
 
+const FOUNDATION_PAGES = [
+  {
+    label: 'Colors',
+    path: '/foundations/colors/',
+  },
+  {
+    label: 'Elevation',
+    path: '/foundations/elevation/',
+  },
+  {
+    label: 'Typography',
+    path: '/foundations/typography/',
+  },
+  {
+    label: 'Layout',
+    path: '/foundations/layout/',
+  },
+  {
+    label: 'Spacing',
+    path: '/foundations/spacing/',
+  },
+  {
+    label: 'Icons',
+    path: '/foundations/icons/',
+  },
+  {
+    label: 'CSS-Utilities',
```

Comment:
I think we don't need `-` in labels (on current prod version the label is `CSS Utilities`), same with `Brand-icons`

Reply:
Corrected, thx!